flang/include/flang/Lower/AbstractConverter.h

```diff
@@ -70,7 +70,7 @@ class AbstractConverter {
   virtual mlir::Value getSymbolAddress(SymbolRef sym) = 0;
 
   /// Bind the symbol to an mlir value.
-  virtual void bindSymbol(const SymbolRef sym, mlir::Value val) = 0;
+  virtual bool bindSymbol(const SymbolRef sym, mlir::Value val) = 0;
```

Comment:
The comment could be updated to tell that if the symbol already has a binding, the function will do nothing and return false.

Reply:
Done.